lib/project_types/php/messages/messages.rb

```diff
@@ -4,8 +4,17 @@ module PHP
   module Messages
     MESSAGES = {
       php: {
+        help: <<~HELP,
+          Suite of commands for developing PHP apps. See {{command:%1$s php <command> --help}} for usage of each command.
```

Comment:
Should we highlight that we're scaffolding a Laravel PHP app, vs. Symfony, etc.?